caom2-compute/src/main/java/ca/nrc/cadc/caom2/compute/PolygonUtil.java

```diff
@@ -105,15 +106,15 @@
      */
     public static final boolean ENABLE_CONCAVE_OUTER = false;
 
-    public static Polygon toPolygon(Shape s)
+    public static MultiPolygon toMultiPolygon(Shape s)
```

Comment:
Most of that code is old and ugly and for the most part deprecated as we move to not computing stuff internally so I'm not super inclined to fix it. We'll nede to keep some of it to support  computations in datalink and cutout services but most will go (I hope).